docs/devices-api/more-info/cordova.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Cordova"
+description: "A plugin to integrate with Metriport on Cordova - including Apple Health integrations."
+---
+
+<Info>
+  Before getting started, visit the [Apple Health
+  docs](/devices-api/more-info/apple-health#1-enable-healthkit) and follow the Enable HealthKit
+  section to configure HealthKit."
+</Info>
+
+## Installation
+
+```sh
+cordova plugin add https://github.com/metriport/cordova-sdk
+```
+
+You'll need to ensure that the MetriportSDK pod is installed as well.
+
+```sh
+cd platforms/ios
+pod install
```

Comment:
they won't have to `pod install` a specific pod?

Reply:
No it works the same as react native where you install the package (or in this case plugin) and then after you run `pod install` in ios itll just see which pods to download based off of what it installed